### PR TITLE
Update errors.dd

### PR DESCRIPTION
**Pull request by smcmurray**
I don't understand how the entire section of error handling never mentions try-catch or how to throw exceptions.

### CODE REVIEW

**Comment by smcmurray** [comment]
This is not much better. It would be nice if somebody could fill in more about try/catch/finally, and maybe some examples using assert().

File: errors.dd [hunk 1]
```diff
@@ -173,6 +173,10 @@ $(DL
 Using exceptions to handle errors leads to another issue - how to write
 exception safe programs. $(DPLLINK exception-safe.html, Here's how).
 
+D uses $(DPLLINK statement.html#TryStatement, try, catch, finally statements)
+to handle exceptions. You can raise Exceptions using throw, as in
+throw new Exception("oops!");
```
Comment (ntrel) [inline errors.dd]: Could wrap this statement in backticks for code formatting.

Comment (smcmurray) [inline errors.dd]: I was hoping somebody who knows D would replace it with something much more
helpful.

**Comment by MartinNowak** [comment]
Well it already cross-references http://dlang.org/statement.html#TryStatement, so you just need to find a nice transition to redirect people there.

File: errors.dd (hunk 1 above)
Comment (MartinNowak) [inline errors.dd]: A basic try/catch/throw paragraph should be at the beginning not at the bottom of this text.

**Comment by MartinNowak** [comment]
At best you expand the existing paragraph.
```
The solution is to use exception handling to report errors. All errors are objects derived from abstract class Error. Error has a pure virtual function called toString() which produces a char[] with a human readable description of the error.
```

**Comment by MartinNowak** [comment]
> I don't understand how the entire section of error handling never mentions try-catch or how to throw exceptions.

The page is pretty old already.

**Comment by andralex** [comment]
@smcmurray ping

**Comment by smcmurray** [comment]
@andralex ?

**Comment by andralex** [comment]
I'll close this because it doesn't fix the larger issues; created https://issues.dlang.org/show_bug.cgi?id=15521.

**Pull request closed without merging**